charts/stable/maddy/values.yaml

```
@@ -0,0 +1,168 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: foxcpp/maddy
+  # -- image tag
+  tag: v0.5.2
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```

In new charts we are trying to define the appVersion only once, that way we "force" ourselves to update the appVersion in Chart.yaml as well 😅